Examples/SnapinsSDKKotlinExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/activities/VersionActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.salesforce.snapinssdkexample.activities;
+
+import android.os.Bundle;
+import android.os.PersistableBundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import com.salesforce.android.chat.core.BuildConfig;;
+import com.salesforce.snapinssdkexample.R;
+
+/**
+ * Activity to display the versions of the snap-ins SDKs
+ */
+public class VersionActivity extends AppCompatActivity {
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState, @Nullable PersistableBundle persistentState) {
```

Comment:
This probably isn't being invoked because it's the wrong `onCreate()`. Remove the `@Nullable PersistableBundle persistentState` argument and this activity should start working properly.

Reply:
Done